usds/justice40-tool
# Spanish changes from translation team

# Purpose
- closes #930 

## QA 

### QA link [here](http://usds-geoplatform-justice40-website.s3-website-us-east-1.amazonaws.com/justice40-tool/1650-b5a212/es/#3/33.47/-97.5)

### QA Spec
- [x] On the About page: “Email” can be rendered in Spanish as “email” or as “correo electrónico”. For consistency with the term used in our translation, we recommend using “correo electrónico”, so this section reads “Correo electrónico: screeningtool.feedback@usds.gov”
- Links that are routed to English sites (6 occurrences) should have a prefix of [en ingles]

- [x] Footer link - FOIA
- [x] Footer link - RFI link
- [x] About page - paragraph 1 - Executive order link 1 
- [x] About page - paragraph 2 - Executive order link 2
- [x] About page - paragraph 3 - FAQ link
- [x] Link below the map

- [x] Percentile suffix of `th` in 65th percentile is replaced with `.a` in all occurrences. This is what the translation team mentioned:
```
For the census blocks: All other information in this section is translated into Spanish, 
but the ordinal numbers (highlighted in the screenshot) throughout the tool are 
abbreviated in English (e.g., 19th, 65th). We suggest changing this to the Spanish 
ordinal number abbreviation, which is .o for masculine and .a for feminine gendered 
words. ***Since this ranking applies to the communities of focus, which use a feminine 
gender in Spanish, we recommend that the th ordinal abbreviation in English be 
substituted with the feminine ordinal abbreviation in Spanish: .a  throughout the text. 
E.g., 19th would be 19.a in Spanish and 65th would be 65.a
```
I'm interpreting this to mean that all percentiles will have a `.a` suffix since they are referring to a feminine topic (communities of focus). Not sure if that is different now that it's `disadvantaged`? If so, we can easily change it to `.o`.  Interesting to learn that the suffix is not based on the number, but rather what the number represents. 🤔 
- [x] When a tract is selected, the methodology version on the top of the side panel is in Spanish.
